posthog/temporal/batch_exports/backfill_batch_export.py

```diff
@@ -228,9 +229,10 @@ async def backfill_schedule(inputs: BackfillScheduleInputs) -> None:
                 await sync_to_async(unpause_batch_export)(client, inputs.schedule_id)
                 return
 
+            assert isinstance(description.schedule.action, temporalio.client.ScheduleActionStartWorkflow)
             schedule_action: temporalio.client.ScheduleActionStartWorkflow = description.schedule.action
 
-            search_attributes = [
+            search_attributes: collections.abc.Sequence[temporalio.common.SearchAttributePair[typing.Any]] = [
                 temporalio.common.SearchAttributePair(
```

Comment:
I don't understand why mypy is complaining about these two now, but nothing should be changing here.